res/lang/production/production_en.txt

```diff
@@ -136,15 +136,15 @@ SELECT_PERSON      = Select person
 FINISHED_PLANNING  = Finished planning
 FINISH_PLANNING    = Finish planning
 AND_PAY_DOWN_MONEY = and pay down %money%
-PLANNING           = Planning ...
+PLANNING           = Planning...
 MONEY_TO_PAY_DOWN  = %money% to pay down
 TAKE_OVER_SETTINGS = Copy settings
 PRODUCTION_COMPANY = Production company
 EDIT_TEXTS         = Edit texts
 TITLE              = Title
 DESCRIPTION        = Description
 EPISODE            = Episode
-ABBREVIATION_YEARS = Y.
+ABBREVIATION_YEARS = YO
```

Comment:
really YO = Years?

Reply:
Yes, it's quite common for "years old".  To be honest we often don't abbreviate it but if we do, it's YO.